CHANGELOG.md

```diff
@@ -1,6 +1,21 @@
 # Changelog
 
 This file documents any relevant changes.
+## [1.1.0] 2023-11-23
+- fix:
+- chore: support Viur Core 3.5.6
```

Comment:
Why? viur-cli should not support a specific viur-core version. Anyway, please keep correct module notation... *viur-core*, not *Viur Core*...